UnityProject/Assets/Scripts/Core/Editor/Tools/Tools.cs

```diff
@@ -1,343 +1,442 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using Systems.CraftingV2;
 using Mirror;
+using Objects.Wallmounts;
 using UnityEditor;
 using UnityEditor.SceneManagement;
 using UnityEngine;
 using UnityEngine.SceneManagement;
-using Objects.Wallmounts;
+using Util;
 using Object = UnityEngine.Object;
 
 
-public class Tools : Editor
+namespace Core.Editor.Tools
 {
-	class Conn
```

Comment:
This and other changes below were made by adding namespace to the file.

You might ignore them, except the `CheckAndFixCraftingCrossLinks` methods (at the bottom of the file)